### ISSUE TITLE
Inaccuracy in comparisons guide

### ISSUE THREAD

**Issue by samuelpath** [vuejs/vuejs.org#2429]
I think that issue #668 (Add comparison to Svelte) could be reopened: https://github.com/vuejs/vuejs.org/issues/668

On 10 Jul 2017 @chrisvfritz said: "Closing for now, as Svelte might still be a little too niche to spend time on a comparison"

I believe that this is not the case anymore, given that it ranks #1 in front-end frameworks interest in the 2019 state of JS survey: https://2019.stateofjs.com/front-end-frameworks/

**Comment by phanan** [comment]
I agree. We may want to wait a bit for Vue 3 though, as it will come with features and improvements that are more up to date in comparison with Svelte (as well as other frameworks).

**Comment by Charuru** [comment]
Really want to see this. Just selfishly, can somebody get started on it with Vue3? I'd really like to know.
